proto/xudp2p.proto

```diff
@@ -18,6 +18,7 @@ message Order {
 message Node {
     string node_pub_key = 1;
     repeated Address addresses = 2;
+    bool inbound = 3;
```

Comment:
I think we should be keeping `node.inbound` as internal and not send it within the `NodesPacket`.